cccs-build/superset/Dockerfile

```
@@ -1,7 +1,7 @@
 # Vault CA container import
 ARG VAULT_CA_CONTAINER=uchimera.azurecr.io/cccs/hogwarts/vault-ca:master_11376_a25c34e1
 FROM $VAULT_CA_CONTAINER AS vault_ca
-FROM uchimera.azurecr.io/cccs/superset-base:cccs-2.1_20230726145242_b7395
+FROM uchimera.azurecr.io/cccs/superset-base:bug--Multiple-adhoc-dashboard-filters_20230822131759_b7516
```

```suggestion
FROM uchimera.azurecr.io/cccs/superset-base:cccs-2.1_20230726145242_b7395
```

This shouldn't be updated until after the main PR has been merged in (then you can create a second PR to update it).